backend/contextProperties.js

```diff
@@ -35,33 +34,32 @@ const copyNamedHeaders = (headerNames, srcHeaders) =>
     if (srcHeaders[name]) {
       return {
         ...previous,
-        [name]: srcHeaders[name],
+        [name]: parseInt(srcHeaders[name], 10),
```

Comment:
Does this try and convert all headers values into a int?

Reply:
Looks like it mainly handlers the pagination type headers. If so might be worth tweaking the function name a bit.